item-impl/src/main/scala/com/example/auction/item/impl/ItemServiceImpl.scala

```diff
@@ -31,6 +31,14 @@ class ItemServiceImpl(registry: PersistentEntityRegistry, itemRepository: ItemRe
     }
   })
 
+  override def updateItem(id: UUID) = authenticated(userId => ServerServiceCall{ item =>
```

Comment:
Am I missing something, or is the verification that the caller is indeed the creator of the item only performed in the view?